### PR TITLE
Make minor grammar corrections/updates to async/promise-basics

### PR DESCRIPTION
**Pull request by jchue**
Split out and revised changes originally in #1612

### CODE REVIEW

File: 1-js/11-async/02-promise-basics/article.md [hunk 1]
```diff
@@ -31,9 +31,9 @@ When the executor obtains the result, be it soon or late - doesn't matter, it sh
 - `resolve(value)` — if the job finished successfully, with result `value`.
 - `reject(error)` — if an error occurred, `error` is the error object.
 
-So to summarize: the executor runs automatically, it should do a job and then call either `resolve` or `reject`.
+So to summarize: the executor runs automatically. It performs a job and then calls either `resolve` or `reject`.
```
Comment (iliakan) [inline 1-js/11-async/02-promise-basics/article.md]: Is the original variant wrong?

Comment (paroche) [reply 1-js/11-async/02-promise-basics/article.md]: Not very wrong. Probably wouldn't have triggered a PR from me (I hadn't read this article yet), but the changed version is a bit better. However, I would offer another alternative:

"...runs automatically and attempts to do a job. When it is finished with the attempt it calls `resolve` if it was successful or `reject` if there was an error."

If it's not too wordy.

**Comment by jchue** [comment]
Updated

**Comment by iliakan** [comment]
@paroche could you please advise, should I merge this PR or not?

**Comment by paroche** [comment]
I would say yes --  from my review of them just now, looks like all the changes are improvements to a greater or lesser degree. I prefer my suggested wording on line 34, but that could be handled later if desired.

**Comment by iliakan** [comment]
@paroche thanks, merged then!
